src/elfpy/pricing_models.py

```diff
@@ -539,6 +539,58 @@ def calc_out_given_in(
 
         return (without_fee_or_slippage, with_fee, without_fee, fee)
 
+    def calc_lp_out_given_tokens_in(
+        self,
+        d_base,
+        share_reserves,
+        bond_reserves,
+        share_buffer,
+        init_share_price,
+        share_price,
+        lp_reserves,
+        rate,
+        time_remaining,
+        stretched_time_remaining,
+    ):
+        return HyperdrivePricingModel().calc_lp_out_given_tokens_in(
```

Comment:
I'm not sure that these would be the same. If they are, then this function should live in the root class (`PricingModel`).

Reply:
This has been removed